helm/hpcc/templates/_helpers.tpl

```diff
@@ -1477,6 +1480,43 @@ resources:
 {{- end -}}
 {{- end -}}
 
+{{/*
+Add minimal resource allocation for postrun sidecar containers
+
+This template generates low-impact CPU and memory resource specifications
+for utility sidecar containers that perform lifecycle management tasks.
+These containers typically monitor main application containers and perform
+cleanup operations, requiring minimal cluster resources.
+
+The resource values ensure guaranteed scheduling while preventing resource
+contention with primary HPCC Platform workloads (Dali, Thor, Roxie).
+
+Pass in: Current template context (.)
+Returns: YAML resources block with minimal requests and limits
+*/}}
+{{- define "hpcc.addPostrunResources" }}
```

Comment:
Adding this because resource calculations consider attached sidecarts, without these resources, the hpa resources would be "unknown" and max replicas would be created

Reply:
> Let's move the addition of missing resource specs for postrun to a separate PR.

I get the reasoning, but this feature is incomplete without this change and the change is self contained, if the review is too widespread and you absolutely think the benefit of a dedicated PR outweights the effort let's discuss.